src/types/formatted/relatedLinks.ts

```diff
@@ -0,0 +1,10 @@
+export type RelatedLink = {
```

Comment:
This is the shared component that took the place of the `BenefitHubLinks` and `RelatedInformation` since they were essentially the same thing.